About/Keys.md

```diff
@@ -147,7 +147,7 @@ The owner of the machine generates a GPG key pair as part of installing Heads.
   Ideally the private key does not live on the machine, but instead is in a
   Yubikey or other USB Security dongle.
 
-TODO: Can this be used in the disk decryption process?
+The security dongle may be used in the disk decryption process. Some of the Linux distros have incoporated this using /etc/crypttab with a keyscript option.
```

Comment:
incoporated -> incorporated

Example of keyscript https://docs.puri.sm/PureBoot/LibremKeyLUKS.html
https://docs.puri.sm/Librem_Key/Getting_Started/User_Manual.html#decrypt-luks-encrypted-drives-with-librem-key

You can change Linux distros to point to first link so users interested can dig on their own.